src/main/java/com/google/neighborgood/servlets/UserInfoServlet.java

```diff
@@ -38,14 +39,53 @@
 public class UserInfoServlet extends HttpServlet {
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     UserService userService = UserServiceFactory.getUserService();
 
+    // Retrieves user accounts for the topscorers board
+    if (request.getParameterMap().containsKey("action")
```

Comment:
Am I reading this correctly that users don't need to be logged in to view the top 10 list?  Is that what we want?

Reply:
I had actually thought of allowing guest users to see the top10 lists as well - do you think this is a bad idea? I can change this to be only visible to logged users if that's the case.

Reply:
I like the idea!  My only thought is about how this performs a query/sort of the entire UserInfo table which sounds like it could slow down the server if lots of random users load this page.  I think a typical pattern for this problem is to have a background task on the server that periodically performs this computation and then writes the output to a "top 10" table in the database, and then external users loading the "top 10" page will read from this "top 10" table.  I have no idea how easy/hard this is to do with this server framework, so I'll let you decide if this is reasonable or not.  :-)

Reply:
This is actually a really good idea, thanks! I'm not 100% certain how to implement this either, but I'll look into it after the MVP presentation and add it to our list of 'issues' so I can hopefully address this in an upcoming PR :)